content/docs/manual/tls.md

```diff
@@ -233,3 +201,41 @@ $ zn_pub -c peer.json5 -l tls/localhost:7448 -e tls/localhost:7447
 ```
 
 As it can be noticed, the same *peer.json5* is used for *zn_sub* and *zn_pub*.
+
+---------
+## Appendix: TLS certificates creation
```

Comment:
Did you try to use `Let's encrypt` to generate the certificate? That would make all clients work when a router uses a certificate generate by  `Let's encrypt`.

Reply:
No I didn't manage yet, I may be too inexperienced regarding how LetsEncrypt works but the certificate creation process requires us to specify a domain name (it seems like it's too web oriented), after what I'll have to solve some "http challenges" to certify I own the "website". 
I was planning on continuing investigating how to get certificates, but for the moment I thought it was convenient to clarify the documentation regarding the default behaviour of the TLS feature.